core/src/main/java/org/testcontainers/containers/Network.java

```diff
@@ -16,6 +16,8 @@
 
 public interface Network extends AutoCloseable, TestRule {
 
+    Network SHARED = newNetwork();
```

Comment:
since "network alias" functionality requires Network to be set, I created a `SHARED` network for different use cases (to replace the links)

Reply:
Cool, we might be able to use it for a more modern Docker-Compose implementation as well?